acousticbrainz/views/data.py

```diff
@@ -44,21 +45,44 @@ def view_high_level(mbid):
 
 @data_bp.route("/<uuid:mbid>", methods=["GET"])
 def summary(mbid):
-    lowlevel, highlevel, genres, moods, other = get_summary_data(mbid)
-
-    info = _get_track_info(mbid, lowlevel['metadata'])
+    summary = get_summary_data(mbid)
+    
+    lowlevel = None
+    highlevel = None
+    genres = None
+    moods = None
+    other = None
+    if 'lowlevel' in summary:
+        lowlevel = summary['lowlevel']
+    if 'highlevel' in summary:
+        highlevel = summary['highlevel']
+    if 'genres' in summary:
+        genres = summary['genres']
+    if 'moods' in summary:
+        moods = summary['moods']
+    if 'other' in summary:
+        other = summary['other']
```

Comment:
Code above can be converted to one-liners which would make it easier to read. For example:

```python
lowlevel = summary['lowlevel'] if 'lowlevel' in summary else None
```

Alternatively (which might be a better option), you can modify ``get_summary_data`` function to return ``None`` in dictionary values if data is missing. That way you won't even need to check if these values are defined.